lib/android-helpers.js

```diff
@@ -665,12 +665,13 @@ helpers.unlock = async function unlock (driver, adb, capabilities) {
     await helpers.unlockWithHelperApp(adb);
   } else {
     await helpers.unlockWithUIAutomation(driver, adb, {unlockType: capabilities.unlockType, unlockKey: capabilities.unlockKey});
-    await helpers.verifyUnlock(adb);
+    await helpers.verifyUnlock(adb, capabilities.unlockSuccessTimeout);
```

Comment:
Please add the new capability to https://github.com/appium/appium-android-driver/blob/master/lib/desired-caps.js